### DISCUSSION TITLE
Exporting issue

### DISCUSSION THREAD

**Discussion by BalasIgor1998** [Parik27/DragonFF#225] [Q&A] [answered]
I can't figure out what is hierarchy for modeling?
What I've done is added modified wheel under wheel_rf_dummy called wheel and exported as dff in game and game crash. So I would like to know is blender able to make hierarchy or some other program should do it??

**Comment by BalasIgor1998** [comment]
So i deleted old wheel mesh and added new one as that name, "wheel" added modifier edge split as game requested, and not working

**Comment by ghost** [comment]
So you are exporting the model incorrectly

**Comment by BalasIgor1998** [comment]
But how?, after editing model and checking hierarchy i go File - export dragondff export as dff and it usually have allready name of car called example admiral.dff

**Comment by BalasIgor1998** [comment]
And I export it in modloader but i dont think so it is problem

**Comment by BalasIgor1998** [comment]
For example.
I see that people in mods using their own name under mesh that in original have different name and they adding extra mesh in some other mesh and that works fine so im really confused how to set that up

**Comment by BalasIgor1998** [comment]
So If I join my model with original wheel mesh and just delete original wheel faces mod works but if i add my model delete original and in wheel dummy add my model and rename it in wheel game not working i just dont get it

**Comment by ghost** [comment]
Are you attaching a new wheel to wheel_dummy?

**Comment by Parik27** [comment]
This isn't the place for help with making models for any game. Only post issues with the add-on itself. Use discussions or GTAForums for questions such as these.

**Comment by Psycrow101** [comment, marked as answer]
In the latest version of DragonFF you can change the order of frames. Using the "Dragon FF - Frames" panel in the scene properties, change the position of your created object to match the original position.

Related issues:
https://github.com/Parik27/DragonFF/issues/334
https://github.com/Parik27/DragonFF/issues/312
